### PR TITLE
remove cassim from build until fixed

### PR DESCRIPTION
**Pull request by janl**
needs https://github.com/apache/couchdb-chttpd/pull/119

### CODE REVIEW

File: rel/reltool.config [hunk 1]
```diff
@@ -87,8 +86,6 @@
     %% couchdb
     {app, b64url, [{incl_cond, include}]},
     {app, bear, [{incl_cond, include}]},
-    {app, cassim, [{incl_cond, include}]},
-    {app, chttpd, [{incl_cond, include}]},
```
Comment (kxepal) [inline rel/reltool.config]: Why drop chttpd?

Comment (rnewson) [inline rel/reltool.config]: yeah, that's an oops.

Comment (janl) [inline rel/reltool.config]: haha, editor key foobar :D

Comment (janl) [inline rel/reltool.config]: fixed in latest push

**Comment by rnewson** [comment]
+1

**Comment by kxepal** [comment]
+1

**Comment by kxepal** [comment]
);

**Pull request merged**